### PR TITLE
Feat/add draft ta liste des courses avec api plateform

### PR DESCRIPTION
**Pull request by vdelpeyroux**
just pour mon draft et les asset

### CODE REVIEW

**Comment by ElevenWilson** [comment]
51 words might be misspelled, please check them: apidoc, APIs, backend, Backend, baseurl, Behat, Clonons, consolegen, d'API, démo, docapi, FOSuser, fpm, GraphiQL, graphql, graphQL, Graphql, GraphQL, gretting, http, js, JSON, JWT, Kubernites, lang, l'autentification, LD, listcourse, listecourse, l'url, NelmioApiDoc, Nginx, PHPUnit, Pierlot, Plateform, Platfrom, png, poper, PostgreSQL, redux, Redux, shoppingItem, src, symfony, Symfony, taguée, technos, TLS, vdelpeyroux, workspace, yml

**Comment by ElevenWilson** [comment]
51 words might be misspelled, please check them: apidoc, APIs, backend, Backend, baseurl, Behat, Clonons, consolegen, d'API, démo, docapi, FOSuser, fpm, GraphiQL, graphql, graphQL, Graphql, GraphQL, gretting, http, js, JSON, JWT, Kubernites, lang, l'autentification, LD, listcourse, listecourse, l'url, NelmioApiDoc, Nginx, PHPUnit, Pierlot, Plateform, Platfrom, png, poper, PostgreSQL, redux, Redux, shoppingItem, src, symfony, Symfony, taguée, technos, TLS, vdelpeyroux, workspace, yml

**Comment by ElevenWilson** [comment]
52 words might be misspelled, please check them: apidoc, APIs, backend, Backend, baseurl, Behat, Clonons, consolegen, d'API, débugger, démo, docapi, FOSuser, fpm, GraphiQL, graphql, graphQL, Graphql, GraphQL, gretting, http, js, JSON, JWT, Kubernites, lang, l'autentification, LD, listcourse, listecourse, l'url, NelmioApiDoc, Nginx, PHPUnit, Pierlot, Plateform, Platfrom, png, poper, PostgreSQL, redux, Redux, shoppingItem, src, symfony, Symfony, taguée, technos, TLS, vdelpeyroux, workspace, yml

**Comment by ElevenWilson** [comment]
52 words might be misspelled, please check them: apidoc, APIs, backend, Backend, baseurl, Behat, Clonons, consolegen, d'API, débugger, démo, docapi, FOSuser, fpm, GraphiQL, graphql, graphQL, Graphql, GraphQL, gretting, http, js, JSON, JWT, Kubernites, lang, l'autentification, LD, listcourse, listecourse, l'url, NelmioApiDoc, Nginx, PHPUnit, Pierlot, Plateform, Platfrom, png, poper, PostgreSQL, redux, Redux, shoppingItem, src, symfony, Symfony, taguée, technos, TLS, vdelpeyroux, workspace, yml

File: _posts/fr/2018-04-05-api-platform.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+ 
+Ce framework est vraiment des plus complets. Ce que l'on vient d'aborder ne couvre même pas le quart de ce que propose API Platform dans sa globalité, et toutes ces possibilités se basent sur de bonnes pratiques.
+
+Le seul point négatif que l'on peut lui trouver c'est son architecture qui est orientée événement, se basant sur le système des events kernel de Symfony. L'avantage est que cela donne une grande liberté pour développer de nouvelles fonctionnalités.  
+
+Cela devient un inconvénient sur de gros projet où l'historique du code produit par moult développeurs, sûrement très bien attentionnés, devient un peu trop conséquent. Cela peut très vite partir dans tout les sens, et au final être un enfer à débugger.
+
+## À venir
+  
+J'aurais aimé vous parler de l'intégration de FOSuser bundle avec l'autentification JWT, des tests en Behat, PHPUnit, déployer son projet API Platform sur un Kubernites cluster en quelques lignes de commande mais le temps me manque, je vous réserve donc cela pour la prochaine fois.
```
Comment (flemzord) [inline _posts/fr/2018-04-05-api-platform.md]: son projet API Platform sur un **cluster Kubernetes** en quelques lignes

Comment (flemzord) [reply _posts/fr/2018-04-05-api-platform.md]: @vdelpeyroux

**Pull request merged**